crates/bevy_ui/src/geometry.rs

```diff
@@ -176,20 +176,42 @@ impl UiRect {
     /// assert_eq!(ui_rect.top, Val::Px(10.0));
     /// assert_eq!(ui_rect.bottom, Val::Px(10.0));
     /// ```
-    pub fn all(value: Val) -> Self {
+    pub const fn all(value: Val) -> Self {
         UiRect {
             left: value,
             right: value,
             top: value,
             bottom: value,
         }
     }
+
+    /// ```rust
+    /// use bevy_ui::UiRect;
+    /// assert_eq!(UiRect::DEFAULT, UiRect::default());
+    /// ```
+    pub const DEFAULT: UiRect = UiRect::UNDEFINED;
```

Comment:
I don't see the use for `DEFAULT` if we already have `default()`.

Reply:
Unfortunately the default trait's method is not usable in const contexts currently. It also produces less helpful docs, as the values are not shown on hover.

I'm fine to cut this from this PR though.

Reply:
We should probably cut this from this PR to reduce controversy.

Reply:
Having a `DEFAULT` alias here isn't too painful imo, as it clarifies what should be used by default.

Perhaps a docs would be nice to have? I would suggest:
```rust
/// Alias of [`UiRect::UNDEFINED`]
```